test/integration/simple_example/controls/forseti.rb

```diff
@@ -0,0 +1,156 @@
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+title 'Forseti Terraform GCP Test Suite'
+
+project_id = attribute('project_id')
+forseti_client_vm_name = attribute('forseti-client-vm-name')
+forseti_server_vm_name = attribute('forseti-server-vm-name')
+forseti_client_gcs_bucket = attribute('forseti-client-gcs-bucket')
+forseti_server_gcs_bucket = attribute('forseti-server-gcs-bucket')
+
+control 'forseti-client-vm' do
+  impact 1.0
+  title 'Test forseti client vm'
+  describe google_compute_instance(
+    project: project_id,
+    zone: 'us-central1-c',
+    name: forseti_client_vm_name
+  ) do
+    it { should exist }
+    its('machine_size') { should eq 'n1-standard-2' }
+  end
+end
+
+
+control 'forseti-server-vm' do
+  impact 1.0
+  title 'Test forseti server vm'
+  describe google_compute_instance(
+    project: project_id,
+    zone: 'us-central1-c',
+    name: forseti_server_vm_name
+  ) do
+    it { should exist }
+    its('machine_size') { should eq 'n1-standard-2' }
+  end
+end
+
+control 'forseti-cloudsql-instance' do
+  impact 1.0
+  title 'Test CloudSQL Instance is up'
+  describe google_sql_database_instances(project: project_id) do
+    its('instance_names') { should include /forseti-server-db-*/ }
+  end
+end
+
+
+control 'forseti-server-iam-roles' do
+
+  impact 1.0
+  title 'Test Server Project IAM Role bindings'
+  describe google_project_iam_binding(project: project_id, role: "roles/storage.objectViewer") do
+    it { should exist }
+    its ('members'){ should include /forseti/ }
+  end
+  describe google_project_iam_binding(project: project_id, role: "roles/storage.objectCreator") do
+    it { should exist }
+    its ('members'){ should include /forseti/ }
+  end
+  describe google_project_iam_binding(project: project_id, role: "roles/cloudsql.client") do
+    it { should exist }
+    its ('members'){ should include /forseti/ }
+  end
+  describe google_project_iam_binding(project: project_id, role: "roles/cloudtrace.agent") do
+    it { should exist }
+    its ('members'){ should include /forseti/ }
+  end
+  describe google_project_iam_binding(project: project_id, role: "roles/logging.logWriter") do
+    it { should exist }
+    its ('members'){ should include /forseti/ }
+  end
+  describe google_project_iam_binding(project: project_id, role: "roles/iam.serviceAccountTokenCreator") do
+    it { should exist }
+    its ('members'){ should include /forseti/ }
+  end
+end
+
+# @TODO Need to figure out the Org level permissions to execute this test
```

Comment:
TODOs should be moved to issues.

Reply:
Dropped in https://github.com/terraform-google-modules/terraform-google-forseti/pull/33/commits/69e3dbf3b77ce842017a82d629c135adfb48b00b and a matching ticket has been created to add an example using the feature this is trying to test.